server_addon/maya/server/settings/publishers.py

```diff
@@ -1047,6 +1056,9 @@ class PublishersModel(BaseSettingsModel):
     "CollectFbxCamera": {
         "enabled": False
     },
+    "CollectFbxModel": {
+        "enabled": False
+    },
```

Comment:
Should we also expose the optional state so that a studio could enforce the FBX generation if they want to?